ESP8266/ESP8266.cpp

```diff
@@ -425,6 +456,56 @@ void ESP8266::_connect_error_handler()
     }
 }
 
+void ESP8266::_no_more_data(const int id)
+{
+    static const uint32_t amount = 0;
+
+    struct packet* packet = (struct packet*) (malloc(
+            sizeof(struct packet) + amount));
+
+    if (!packet) {
+        return;
```

Comment:
I would like to see `tr_error()` here. This might otherwise go unnoticed.

Reply:
Ok, I'll add